src/benchmarks/micro/libraries/System.Diagnostics/Perf_Process.cs

```diff
@@ -15,7 +15,14 @@ public class Perf_Process
         
         [Benchmark]
         public void GetCurrentProcess() => Process.GetCurrentProcess().Dispose();
-        
+
+        [Benchmark]
+        public string GetCurrentProcessName()
+        {
+            using var process = Process.GetCurrentProcess();
```

Comment:
should we be including the cost of Dispose()? it does do some work but perhaps it's not significant.

although I see the test above is doing that too.

Reply:
> should we be including the cost of Dispose()?

I am afraid that we don't have an alternative (due to BDN design). The good thing is that this benchmark is dominated by the sys-call performed by `Process.ProcessName` so the cost of `Dispose` should be negligible.